sessions/forecasting-concepts.qmd

```diff
@@ -48,15 +54,6 @@ options(cmdstanr_print_line_numbers = TRUE)
 
 :::
 
-# Introduction to forecasting as an epidemiological problem, and its relationship with nowcasting and R estimation
-
-See the lecture slides.
-
-::: {.callout-tip}
-Proper scoring rules are scoring rules that make sure no model can get better scores than the *true* model, i.e. the model used to generate the data.
```

Comment:
We need to keep the proper scoring rules definition o think. Ideally we have we are going to use proper scoring rules in the main text and then people can read this to find out what they are

Reply:
I agree we keep it, I just shifted it below (line 373) when we start to introduce evaluation. It seemed odd to have it as the first thing in the session